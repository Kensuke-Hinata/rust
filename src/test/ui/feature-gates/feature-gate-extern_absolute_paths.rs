use core::default;

fn main() {
    let _: u8 = ::core::default::Default();
    //~^ ERROR expected function, found trait `core::default::Default`
}
