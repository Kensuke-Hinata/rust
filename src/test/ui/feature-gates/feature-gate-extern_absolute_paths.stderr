error[E0423]: expected function, found trait `core::default::Default`
  --> $DIR/feature-gate-extern_absolute_paths.rs:4:17
   |
LL |     let _: u8 = ::core::default::Default();
   |                 ^^^^^^^^^^^^^^^^^^^^^^^^ not a function

error: aborting due to previous error

For more information about this error, try `rustc --explain E0423`.
